rpc/rpc.go

```diff
@@ -130,10 +130,7 @@ type ErrorReporter interface {
 // Closing the connection will close the transport and release the bootstrap
 // client provided in opts.
 //
-// Passing nil for opts is the same as passing the zero value. This can be useful
-// for obtaining a reference to the remote side's bootstrap. This in turn allows
-// this listening connection to be used as the RPC client and make calls to the remote
-// side, since capnp connections are symmetric.
+// Passing nil for opts is the same as passing the zero value.
```

Comment:
Nit:  I know what you mean, but I worry this might be a bit cryptic to a newbie.  How about something like:

> If opts == nil, sensible defaults are used.  See Options.

?

Reply:
Sounds good.